### PR TITLE
Add full baseline cfg, fix acceptance rate, use logprobs for n gram logits

### PR DESCRIPTION
**Pull request by michaelpginn**
- Adds a full config over languages, draft models, and gamma values for baseline run
- Fixes acceptance rate following Sandler et al (2025) -- only verified tokens are counted in denominator
- Provides logprobs for ngram model logits, so that softmax works as expected

Sorry for a bit of a messy PR

### CODE REVIEW

**Review by paudelnirajan — APPROVED** [review]
maybe in speedup calculation, we can handle scenario when acceptance rate is 1?

everything else is good to go!
